core/src/main/java/feign/FeignException.java

```diff
@@ -131,6 +136,11 @@ public static FeignException errorStatus(String methodKey, Response response) {
     } catch (IOException ignored) { // NOPMD
     }
 
+    String message = new FeignExceptionMessageBuilder()
+        .withResponse(response)
+        .withRequestMethod(response.request().httpMethod())
```

Comment:
could read the method from the response, eliminating the need to call `withRequestMethod` method

Reply:
fixed